app/lib/frontend/templates/admin.dart

```diff
@@ -64,7 +64,7 @@ String renderAccountPackagesPage({
     headerHtml: _accountDetailHeader(user, userSessionData),
     tabs: [
       Tab.withContent(
-          id: 'packages', title: 'My packages', contentHtml: tabContent),
+          id: 'my-packages', title: 'My packages', contentHtml: tabContent),
```

Comment:
Will this change break existing links to that tab?

Reply:
Perhaps just let the Tab.withContent take another parameter telling if it's private?

Reply:
> Will this change break existing links to that tab?

No, tabs are no longer referenced by their ids, and these are private (authenticated) URLs anyway.